cypress/support/fragments/orders/ordersHelper.js

```diff
@@ -1,3 +1,15 @@
+import { Section, KeyValue } from '../../../../interactors';
+import dateTools from '../../utils/dateTools';
+
 export default {
-  mainLibraryLocation: 'Main Library'
+  mainLibraryLocation: 'Main Library',
+
+  verifyOrderDateOpened:() => {
+    cy.do(Section({ id:'purchaseOrder' }).find(KeyValue('Date opened')).perform(element => {
+      const rawDate = element.innerText;
+      const parsedDate = Date.parse(rawDate.match(/\d{1,2}\/\d{1,2}\/\d{4},\s\d{1,2}:\d{1,2}\s\w{2}/gm)[0]);
+      // For local run it needs to add 18000000
```

Comment:
comment is not clear

Reply:
rewrite comment

Reply:
could you update comment into this place too?
![image](https://user-images.githubusercontent.com/93538670/173107534-b03e70c6-b63e-4b36-8da7-4aa4f872738f.png)

Reply:
Done